examples/mediadevices/src/helpers.js

```diff
@@ -14,42 +15,54 @@ function getDevicesOfKind(deviceInfos, kind) {
   });
 }
 
+function switchLocalTracks(room, newLockTrack) {
```

Comment:
@makarandp0 I would make this simpler:
```js
/**
 * Replace the existing LocalAudioTrack or LocalVideoTrack with
 * a new one in the Room.
 * @param {Room} room - The Room you have joined
 * @param {LocalAudioTrack|LocalVideoTrack} track - The LocalTrack you want to switch to
 * @returns {void}
 */
function switchLocalTracks(room, track) {
  const localParticipant = room.localParticipant;
  const publication = Array(track.kind === 'audio'
    ? localParticipant.audioTracks.values()
    : localParticipant.videoTracks.values())[0];
  if (publication) {
    publication.track.stop();
    publication.unpublish();
  }
  localParticipant.publishTrack(localTrack);
}
```

Reply:
@manjeshbhargav -  I think current version reads better, and by using track.kind avoid an extra if check. but good point about track.stop, I will update to stop the track before unpublishing.